### ISSUE TITLE
No Way To Review Failed Files

### ISSUE THREAD

**Issue by seniorm0ment** [Catfriend1/syncthing-android#789]
### Description of the issue

There is no way to review files that are stuck/failed in the sync process for whatever reason, unless you enter through the Web GUI.
To be clear by failed files, I mean files that fail to sync due to various reasons such as naming schemes, storage space, etc.
### Reproduction Steps

Wait/force a file to get stuck/failed, the file is now stuck and you cannot view why it is failed unless you go to the web gui (which is a bit tedious on mobile).

### Version Information
- App Version: Syncthing-Fork 1.15.1.0 (F-Droid)
- Syncthing Version: v1.15.1
- Android Version: Android 10
- Device manufacturer: LG
- Device model: v40

### Device platform info

```sh

```
Can't get this atm, I don't think it's needed though but if so lmk I can grab it when I have time.

### Android Log

```sh

```
Can't get this atm, I don't think it's needed though but if so lmk I can grab it when I have time.

**Comment by Catfriend1** [comment]
Yes , can be improved. Currently, we only show critical "big" system failures with a red text in the folder view but don't have an UI to review failed items. This should be easy to do and has to placed in the right place of the screens to not clutter the interface.

**Comment by Catfriend1** [comment]
"There is a way", so the title "No way..." wasn't correct.

**Comment by xeruf** [comment]
Maybe they can be intermediately listen in "Recent changes"? That's where I searched intuitively.
This issue is bugging me right now actually ^^
